src/components/_video/VideoCategoryCard/FeaturedVideoCategoryCard.tsx

```diff
@@ -48,21 +50,14 @@ export const FeaturedVideoCategoryCard: React.FC<FeaturedVideoCategoryCardProps>
         classNames={transitions.names.fade}
       >
         <FeaturedContainer
-          to={absoluteRoutes.viewer.category(id)}
+          to={categoryUrl}
           ref={hoverRef}
           isLoading={isLoading}
           variantCategory={variant}
           color={color}
         >
           <PlayerContainer>
-            {
-              <BackgroundVideoPlayer
-                src={isLoading ? undefined : videoUrl}
-                loop
-                muted
-                playing={variant === 'default' ? isVideoHovering : true}
-              />
-            }
+            {<BackgroundVideoPlayer src={isLoading ? undefined : videoUrl} loop muted playing={isPlaying} />}
```

Comment:
```suggestion
            <BackgroundVideoPlayer src={isLoading ? undefined : videoUrl} loop muted playing={isPlaying} />
```
We don't need curly braces here